skills_ml/algorithms/occupation_classifiers/classifiers.py

```diff
@@ -2,22 +2,32 @@
 import logging
 import json
 import tempfile
+import yaml
+import boto
 
 from collections import Counter
 
 from gensim.models import Doc2Vec
 
 from skills_ml.algorithms.string_cleaners import NLPTransforms
 
-from skills_utils.s3 import download
+from skills_utils.s3 import download, split_s3_path
 
-PATHTOMODEL = 'open-skills-private/model_cache/'
-LOOKUP = 'lookup_va_0605.json'
+with open('config.yaml') as f:
```

Comment:
I actually don't think we want to introduce any config parsing in skills-ml. It's going to be cleaner to handle the config in one place (skills-airflow) and pass in those variables to skills-ml objects.

So the model cache s3 prefix can just be an argument sent to the constructor.